CONTRIBUTING.md

```diff
@@ -69,16 +69,16 @@ second to sign our CLA, which can be found
 
 Installing Go
 -------------
-InfluxDB requires Go 1.7.4.
+InfluxDB requires Go 1.7.5.
```

Comment:
I know the wording pre-dates this change, but perhaps we should change it to "recommends" instead.